src/controllers/controller.js

```diff
@@ -0,0 +1,62 @@
+import User from '../models/model';
+
+class UsersController {
+  constructor() {
+    this.users = [];
+  }
+
+  createUser(login, password, age) {
+    const newUser = new User(login, password, age);
+    this.users.push(newUser);
+    return newUser;
+  }
+
+  showAllUsers() {
+    return this.users;
```

Comment:
When no users - returns `No users found.`, but should return JSON error

Reply:
Updated status codes and returned JSON